adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -152,8 +167,27 @@ private void validateAcquireTokenRequest(final AuthenticationRequest authenticat
         }
 
         // validate authority
+        Telemetry.getInstance().startEvent(authenticationRequest.getTelemetryRequestId(),
+                EventStrings.AUTHORITY_VALIDATION_EVENT);
+        APIEvent apiEvent = new APIEvent(EventStrings.AUTHORITY_VALIDATION_EVENT);
```

Comment:
file an issue for what?

---
In reply to: [92570212](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r92570212) [](ancestors = 92570212)